spec/features/staff/users_can_create_a_third_party_project_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.feature "Users can create a project" do
+  let(:beis) { create(:delivery_partner_organisation) }
```

Comment:
This probably shouldn't be called 'beis'? I don't think we actually even use this?

Reply:
Aaah, the old cut & paste 😳